packages/atomic/src/components/atomic-result-list/replace-breakpoint.ts

```diff
@@ -0,0 +1,47 @@
+function replaceMediaQuery(style: string, breakpoint: number) {
+  return style.replace(
+    /\(min-width: 1024px\)/g,
```

Comment:
Instead of targeting `1024px`, would it be more robust to use a placeholder/variable name in `atomic-result-list.pcss` ?